### DISCUSSION TITLE
Looking for a Zigbee controller to make light bulb smart

### DISCUSSION THREAD

**Discussion by PatBro12** [Koenkk/zigbee2mqtt#12177] [Devices]
Hello,

I'm looking for a Zigbee controller to make a standard filament light bulb smart.
After this, I want to use the Philips Hue Wall Switch module to switch on/off and dim the lights.

I want do this in that way, because I'm free to place the normal in-wall switch anywhere I want.
What I have found so far are Zigbee controllers to use as switch as well. The problem here is they need wiring from light to controller (light bulb directly connected to that controller). In my situation there is no direct connection from light to switch.

Ideal situation is to switch the light as a normal Philips Hue light with Philips Wall Switch in any place I want.

Thanks.

**Comment by lux73** [comment]
why not use a Hue Bridge?

all other Solutions need a Zigbee Coordinator, Hardware to Connect via USB (like a Raspberry) with OS Software installed like Linux - it seems that is really overcomplicated to Switch a Hue Lamp on/off with an ordinary Hue Wall Switch

z2m is designd for Cross Link different Vendor Hardware with an ordinary Smart Home Software like HomeAssitant, OpenHAB and so on...

**Reply by PatBro12** [reply]
Thanks for the answer.
My plan is to use other Zigbee products in the future. And with z2m I have more possibilities to program the wall switch. Anyway, z2m has more possibilities.

And the filament light want to use now, is not smart yet. It is a Calex titanium giant splash.

**Reply by lux73** [reply]
okay. The Filament is not Smart? then the only possibility is to use an Smart Plug and turn the Filament on/off with these Plug.

for Start you will need this:

- [Zigbee Coordinator](https://www.zigbee2mqtt.io/guide/adapters/#recommended) / please DO NOT USE CHEAP CC2531 - THEY ARE CRAP
- [USB extension Cable](https://www.zigbee2mqtt.io/guide/adapters/#no-logging-is-shown-at-all)
- [Zigbee Smart Plug](https://www.amazon.de/Philips-Hue-smarte-Steckdose-kompatibel/dp/B09CV7QQKB)

I'm using [Slea.sh CC2652RB Coordinator](https://slae.sh/projects/cc2652/) - but be aware and read this Thread: https://github.com/Koenkk/zigbee2mqtt/discussions/6702 - there is sometimes huge Delay with shipping

The [USB extension Cable](https://www.amazon.de/-/en/InLine-34651-USB-2-0-Extension-Cable-Female/dp/B01HVYFP8O) is to avoid Interference and is essential if you want a reliable System

The Zigbee Smart Plug is recommended to take original Philips Hue - some other Vendors have some Problems with Zigbee Routing. So the Zigbee Mesh is not working properly

From Software side you must decide which way you want to go - i recommend Linux OS

**Comment by PatBro12** [comment]
I just bought a Sonoff Zigbee stick and z2m is up and running in Docker on my NAS. I decided to install HA on a Intel NUC. Because a Raspberry Pi is poorly available.
It is a requirement that the light is dimmable. I found a Zigbee dimmer from Robb which can be switched on/off with a pulse switch.
The only disadvantages that I need direct wiring from the light to the switch. And the idea was that I can place the switch on another location. On that location I do not have a direct link to the light.
